components/ToFullscale.coffee

```diff
@@ -54,7 +54,6 @@ tryFindingFullscale = (url, out, callback) ->
   # Verify that it exists
   superagent.head fullUrl
   .end (err, res) ->
-    return callback err if err
```

Comment:
Another option is to use https://github.com/kmalakoff/superagent-ls or keep with superagent `0.21.x`.

@bergie opinions on that? I remember that we already had a conversation about that, maybe with @trustmaster?